.github/workflows/autodocs-images.yaml

```diff
@@ -53,14 +51,19 @@ jobs:
           cp -R "${{ github.workspace }}/${{ env.WORKDIR }}" "${{ github.workspace }}/edu/content/chainguard/chainguard-images" && \
           echo "Finished copy"
 
-      - name: "Get Changelog"
+      - name: "Get Latest Update Log"
         run: |
           EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
           echo "CHANGELOG<<$EOF" >> $GITHUB_ENV
           echo "`cat ${{ github.workspace }}/edu/autodocs/last-update.md`" >> $GITHUB_ENV
           echo "$EOF" >> $GITHUB_ENV
         id: images-build-output
 
+      - name: "Append Update to Changelog"
+        run: |
+          [ ! -f "${{ github.workspace }}/edu/autodocs/changelog.md" ] && touch ${{ github.workspace }}/edu/autodocs/changelog.md
+          sed -i '1s/^/${{ env.CHANGELOG }}\n\n/' ${{ github.workspace }}/edu/autodocs/changelog.md
```

Comment:
It always comes back to sed grep and awk doesn't it 🤣

Reply:
YESSSSS! gosh.